app/views/feedback/new.html.haml

```diff
@@ -0,0 +1,6 @@
+%section.form_wrapper
+  = simple_form_for(@form, url: feedback_path) do |f|
+    = f.hidden_field :url
+    = f.input :email, as: :email
+    = f.input :comment, as: :text
+    = f.button :submit, 'Send Feedback', role: 'button'
```

Comment:
I know. I'm still waiting on content from the content people. I don't know what the H1 should say yet! :( I'll follow up.